staging/src/kubevirt.io/client-go/kubecli/kubevirt.go

```diff
@@ -114,6 +114,10 @@ type StreamOptions struct {
 	Out io.Writer
 }
 
+type VNCOptions struct {
```

Comment:
Why is this structure exported in this file? It seems it is only used to format JSON output in the `vnc.go`. In that case it may be easier to use an anonymous struct.

Reply:
ack, corrected